docs/proposals/1199-multitenant-api-proposal/README.md

```diff
@@ -0,0 +1,250 @@
+  metav1.TypeMeta
+  metav1.ObjectMeta
+
+  Spec InferenceObjectivesSpec
+}
+
+type InferenceObjectivesSpec struct {
+	// Scope: Identifier
+    ID string
+
+	// Scope: Pool binding
+    PoolRef InferenceObjectReference
+
+
+	// Scope: Policy Handles (the following are different options on how policy binding may look)
+
+	// Include core policies as a field of the object
+	Criticality *int
+	FairnessPolicy NotYetDefinedFairnessPolicy
+	SLOPolicy InferenceSLOPolicy
+
+	// or
+
+	// Create an array fields that can bind arbitrary policies (including the 'core' policies)
+	Policies []InferenceObjectReference
+
+	// or 
+
+	// one-of: (Where Policy class would contain all 'core' policies that can be configured.)
+	PolicyProfileRef InferenceObjectReference
+	PolicyProfile PolicyProfile
+}
+
+// PoolObjectReference identifies an API object within the namespace of the
+// referrer.
+type InferenceObjectReference struct {
+	// Group is the group of the referent.
+	//
+	// +optional
+	// +kubebuilder:default="inference.networking.x-k8s.io"
+	Group Group `json:"group,omitempty"`
+
+	// Kind is kind of the referent. For example "InferencePool".
+	//
+	// +optional
+	// +kubebuilder:default="InferencePool"
+	Kind Kind `json:"kind,omitempty"`
+
+	// Name is the name of the referent.
+	//
+	// +kubebuilder:validation:Required
+	Name ObjectName `json:"name"`
+}
+
+
+type PolicyProfile struct {
+	metav1.TypeMeta
+    metav1.ObjectMeta
+
+    Spec PolicyProfileSpec
+}
+
+type PolicyProfileSpec struct {
+	// this is a departure from InferenceModel that used string for criticality.
+	// We got quite a bit of feedback around allowing for custom criticality bands, so an int/enum is more flexible & carries inherent stack rank value.
+	Criticality *int
+	FairnessPolicy NotYetDefinedFairnessPolicy
+	SLOPolicy InferenceSLOPolicy
+}
+
+type InferenceCriticalityPolicy struct {
+    Criticality int32
+}
+```
+
+### Intent
+
+The purpose(s) of the `InferenceObjectives` is:
+- Create a strong concept of a tenant within the inference pool, used to associate groups of requests together for the purpose of Flow Clontrol - which can enforce:
+  - Fair resource sharing
+  - Inter-tenant prioritization
+  - SLO attainment
+- Create a handle with which to attach scheduling policies, allowing for heterogenous scheduling behavior across tenants 
+  - Such as the [InferenceSLOPolicy](https://docs.google.com/document/d/1j2KRAT68_FYxq1iVzG0xVL-DHQhGVUZBqiM22Hd_0hc/edit?resourcekey=0-5cSovS8QcRQNYXj0_kRMiw&tab=t.0#heading=h.emkaixupvf39).
+- Detach identification from the modelName field
+
+## Usage
+
+The InferenceObjectives API surface has 3 general scopes:
+- Identification
+- Pool Binding
+- Policy Handle
+
+### Pool Binding
+
+Tackling the simplest first; the single Pool Binding field `PoolRef` is simply to associate a given InferenceObjectives object with a pool. Meaning that InferenceObjectives with duplicate ID's across different pools are considered valid.
```

Comment:
the objectives would be duplicated with the exception of the pool, correct?
Should we consider making the pool reference an array to avoid the repetition and ensure that policies which are meant to be identical across multiple pools in the namespace do not diverge?